### ISSUE TITLE
Scala plugin cannot run scala code

### ISSUE THREAD

**Issue by garylin** [osxpeppermint/peppermint#49]
I create a simple Test.scala file in my working directory.  When try to run it in peppermint, I am getting the following error:
```
No such file or class on classpath: /Users/foo/workspace/Test
```
Looking at the coffee script for the scala plugin, I think the logic for creating the scala command line is odd:
```
outputFile = $Document.info().path.replace( ".#{ $Document.info().extension }", "");
$Terminal.exec("scala", outputFile);
```
Based on this code, I am guessing it will execute "scala /Users/foo/workspace/Test", which confirms with the errors I see in the console.  Can you suggest a fix?  Also I wonder if there is a way I can fix this locally and override the default scala plugin?

**Comment by osxpeppermint** [comment]
Gary, I'm looking into it right now.

Given that I'm not by any means a Scala guru, could you please give me an example of how a simple Scala program is supposed to be executed? (In the terminal, with normal scala commands, I mean...)

Also, should we take for granted that the Scala executables are to be found at some global folder within the `$PATH`? What is the norm, in terms of Scala apps?

**Comment by osxpeppermint** [comment]
Issue - hopefully - fixed by the (upcoming) 1.5 Beta 9.

![screen shot 2015-05-14 at 12 48 24 pm](https://cloud.githubusercontent.com/assets/9476915/7630566/93fa63f0-fa37-11e4-87a9-134db5b517a0.png)
